src/base/ApplicationECSService.ts

```diff
@@ -346,6 +356,7 @@ export class ApplicationECSService extends Resource {
       memory: this.config.memory.toString(),
       requiresCompatibilities: ['FARGATE'],
       networkMode: 'awsvpc',
+      volume: Object.values(volumes),
```

Comment:
If we'd prefer volume not to be present if it's empty, we could achieve that like this:
```suggestion
      volume: Object.keys(volumes).length > 0 ? Object.values(volumes) : undefined,
```
This seemed cleaner, but I don't have a strong opinion either way.